class-amp-img.php

```diff
@@ -55,16 +56,9 @@ private function filter_attributes( $attributes ) {
 			$name = $attribute['name'];
 			$value = $attribute['value'];
 
-			switch ( $name ) {
-				case 'src':
-				case 'alt':
-				case 'width':
-				case 'height':
-				case 'class':
-					$out[ $name ] = $value;
-					break;
-				default;
-					break;
```

Comment:
The switch is an explicit choice here because it makes it easy to skim through the attributes we're whitelisting and let's us handle attribute special attributes more easily if needed (e.g. if `alt` needed to be manipulated in some way we could just break it out into its own `case`).